docs/index.rst

```diff
@@ -39,6 +39,13 @@ The core features include:
    frontend/choices_methods.md
 
 
+.. toctree::
+   :maxdepth: 1
+   :caption: Native Router
```

Comment:
"native" router is a bit weird. Maybe "Router for Data Parallelism"?

Reply:
Native means that's SGLang official 😂 I stand for these words, and that means in the open source communities, we are all most unique.

Reply:
I also feel we can drop "native" here